example/ndpiSimpleIntegration.c

```diff
@@ -206,54 +208,33 @@ static void free_workflow(struct nDPI_workflow ** const workflow)
     *workflow = NULL;
 }
 
-static char * get_default_pcapdev(char *errbuf)
```

Comment:
Why removing that fn?

Reply:
Probably because I got mixed up with the versions. I started developing on a version released in August, and later tried to realign myself ... I may have missed a change.